### ISSUE TITLE
OCGV: Simplified UI Feature Request

### ISSUE THREAD

**Issue by mivalsten** [PowerShell/GraphicalTools#102]
Hello,
Firstly, thanks for awesome package, I really love it.

I have a feature request for UI modification - Would it be possible to change UI so that when ENTER is pressed on the main window, it works as F9+Accept? Current workflow is quite clunky and slow.
Alternatively, to not break backward compatibility / muscle memory, a switch like `-QuickAccept` could be introduced that modifies this behaviour.

Best Regards,
Greg

**Comment by TylerLeonhardt** [comment]
You know what, @tig did this work and it's in master. We're just waiting for the next release of gui.cs before we can ship it 😄 stay tuned!

**Comment by TylerLeonhardt** [comment]
I'm going to mark this as a duplicate of #95 but really this issue is fixed already.

**Comment by mivalsten** [comment]
Thank you! :)
